Core/GDCore/Extensions/Builtin/AdvancedExtension.cpp

```diff
@@ -18,7 +18,7 @@ void GD_CORE_API BuiltinExtensionsImplementer::ImplementsAdvancedExtension(
           _("Built-in extension providing advanced control features."),
           "Florian Rival",
           "Open source (MIT License)")
-      .SetExtensionHelpPath("" /*TODO: Add a documentation page for this */);
+      .SetExtensionHelpPath("/events/functions/return");
```

Comment:
I would prefer if you can use SetHelpPath on the 3 actions. The extension itself is a mix of things, but should not have a help page pointing to something that is specific like this to only a few actions.
When done, you can actually remove `.SetExtensionHelpPath("" /*TODO: Add a documentation page for this */)` entirely :)